packages/flutter/lib/src/gestures/events.dart

```diff
@@ -1847,6 +1847,9 @@ class _TransformedPointerUpEvent extends _TransformedPointerEvent with _CopyPoin
 ///
 ///  * [Listener.onPointerSignal], which allows callers to be notified of these
 ///    events in a widget tree.
+///  * [PointerSignalResolver], which ensures that these events will only be
```

Comment:
it doesn't ensure that, right? It only provides an opt-in mechanism for cooperating agents to disambiguate an event's target.